benchmarks/cpp/nvfuser/batch_norm_backward.cpp

```diff
@@ -0,0 +1,270 @@
+  auto save_mean = makeContigTensor(1, DataType::Float);
+  auto save_var = makeContigTensor(1, DataType::Float);
+
+  fusion->addInput(input);
+  fusion->addInput(grad_output);
+  fusion->addInput(weight);
+  fusion->addInput(running_mean);
+  fusion->addInput(running_var);
+  fusion->addInput(save_mean);
+  fusion->addInput(save_var);
+
+  if (dtype == DataType::Half) {
+    input = castOp(DataType::Float, input);
+    grad_output = castOp(DataType::Float, grad_output);
+  }
+
+  auto eps_ptr = new Double(kEps);
+
+  auto result = batch_norm_backward(
+      input,
+      grad_output,
+      weight,
+      running_mean,
+      running_var,
+      save_mean,
+      save_var,
+      kTraining,
+      eps_ptr,
+      std::vector<bool>(3, true));
+
+  auto grad_input = result.grad_input;
+  auto grad_weight = result.grad_weight;
+  auto grad_bias = result.grad_bias;
+
+  if (dtype == DataType::Half) {
+    grad_input = castOp(DataType::Half, grad_input);
+    grad_weight = castOp(DataType::Half, grad_weight);
+    grad_bias = castOp(DataType::Half, grad_bias);
+  }
+
+  fusion->addOutput(grad_input);
+  fusion->addOutput(grad_weight);
+  fusion->addOutput(grad_bias);
+}
+
+static void NvFuserScheduler_BatchNorm_BWD(
+    benchmark::State& benchmark_state,
+    FusionExecutorCache* fusion_executor_cache,
+    DataType dtype) {
+  TORCH_INTERNAL_ASSERT(dtype == DataType::Float || dtype == DataType::Half);
+
+  const bool kTraining = true;
+  const float kEps = 1e-5;
+
+  std::vector<int64_t> input_shape{
+      benchmark_state.range(0),
+      benchmark_state.range(1),
+      benchmark_state.range(2),
+      benchmark_state.range(2)};
+
+  at::manual_seed(0);
+  auto options =
+      at::TensorOptions().dtype(data_type_to_aten(dtype)).device(at::kCUDA, 0);
+  auto fp32_options =
+      at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  at::Tensor input = at::randn(input_shape, options);
+  at::Tensor grad_out = at::randn(input_shape, options);
+  at::Tensor weight = at::ones({input_shape[1]}, fp32_options);
+  at::Tensor run_mean = at::zeros({input_shape[1]}, fp32_options);
+  at::Tensor run_var = at::ones({input_shape[1]}, fp32_options);
+  at::Tensor save_mean = at::zeros({input_shape[1]}, fp32_options);
+  at::Tensor save_var = at::ones({input_shape[1]}, fp32_options);
+
+  std::vector<c10::IValue> aten_inputs(
+      {input, grad_out, weight, run_mean, run_var, save_mean, save_var});
+
+  runBenchmarkIterations(benchmark_state, fusion_executor_cache, aten_inputs);
+
+  benchmark_state.SetBytesProcessed(
+      int64_t(benchmark_state.iterations()) *
+      (((3 * input.numel()) * int64_t(dataTypeSize(dtype))) +
+       (run_mean.numel() + run_var.numel() + save_mean.numel() +
+        save_var.numel() + weight.numel()) *
+           int64_t(dataTypeSize(DataType::Float))));
+}
+
+//------------------------------------------------------------------------------
+
+static void Baseline_BatchNorm_BWD(
```

Comment:
Maybe change the function name to 'Baseline_CuDNN_BatchNorm_BWD' to distinguish it from the native_batch_norm functions.